src/scss/settings/_variables.scss

```diff
@@ -83,8 +83,16 @@ $font-weight-headings-preload: 400;
 }
 
 
-//  Global Breakpoints
+//  Global Breakpoints and unit intervals
 // ==========================================================================
+
+$unit-intervals: (
+  'px': 1,
+  'em': 0.063, //0.063ems converts to 1px. We need this as some browsers are very specific with breakpoints
```

Comment:
Probably worth putting the calculation in the comments (as this is based on a base font-size of `16px`).  So `1 / 16 = 0.0625`